cves/kernel/CVE-2021-34556.yml

```diff
@@ -245,10 +245,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Unclear how the vulnerability was discovered. Looked at Linux Kernel CVEs
```

Comment:
Possibly look at reword this section to have more complete sentences. Something along the lines of "After researching the Linux Kernel CVE site and patch descriptions, it is not clear how the vulnerability was discovered."